frontend/src/views/web/challenge/edit-challenge/edit-challenge-evaluation-criteria.html

```diff
@@ -21,7 +21,8 @@
                                      trix-attachment-remove="trixAttachmentRemove(e, editor);"
                                      prevent-trix-file-accept = "true"
                                      placeholder="Evaluation Criteria">
-                          </trix-editor>
+                      </trix-editor>
```

Comment:
Are you sure the formatting (change in indent) is right?

Reply:
it was extra indented , pushed a fix